values/kiali-operator/kiali-operator.gotmpl

```diff
@@ -71,19 +71,18 @@ cr:
         cache_duration: 10
         cache_enabled: true
         cache_expiration: 300
-        in_cluster_url: http://po-prometheus.monitoring:9090
```

Comment:
why is this one removed? they should both exist to allow public domain access

Reply:
It's not a valid flag, see [here](https://github.com/kiali/kiali-operator/blob/v1.39/deploy/kiali/kiali_cr.yaml) as reference

Reply:
@Morriz does it answer your question?